code/modules/research/designs/medical_designs.dm

```diff
@@ -84,6 +84,39 @@
 	build_path = /obj/item/weapon/reagent_containers/glass/beaker/noreact
 	category = list("Medical Designs")
 
+/datum/design/bluespacesyringe
+	name = "Bluespace Syringe"
+	desc = "An advanced syringe that can hold 60 units of chemicals"
+	id = "bluespacesyringe"
+	req_tech = list("bluespace" = 1, "materials" = 2)
```

Comment:
@nullbear they and draw inject 1/3rd, specified on the amount_per_transfer_from_this = 20